.changeset/large-planes-act.md

```diff
@@ -0,0 +1,5 @@
+---
+"nextjs-website": patch
+---
+
```

Comment:
why 2 changesets?

Reply:
I think it's because of this [merge commit](https://github.com/pagopa/developer-portal/pull/725/commits/20ef78bdbb67ecd89e90fc184dea5456a4e88b9f)

Reply:
I think it's because of the dependency (as specified in the description, this PR depends on #724)